uwblueprint/focus-on-nature
# Feat: Create camps list table + page

## Notion ticket link

[Camps List Table](https://www.notion.so/uwblueprintexecs/Camp-List-Camps-Table-62ddc2c21fde4973bc07737206864b01)



## Implementation description
* Create camps list table
* Added functionality to search by camp name, filter by status, navigate by year, and delete a camp



## Steps to test
1. Make sure camp data loads correctly by playing around with filters, navigating through the years, searching by name, etc
2. Try deleting a camp and make sure it deletes from database (can check by refreshing the page - the camp should not appear)



## What should reviewers focus on?
* 


## Checklist

NOTE: I don't have my linter working so if anybody is reviewing can you pls run `yarn fix` in the `frontend` directory pls :')) thank you
